package com.webone.quiosq.itg.response;

import com.fasterxml.jackson.databind.PropertyNamingStrategies;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.Data;

@Data
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
public class StatusPagamentoApiResponse {
    private Long id;
    private String status;
    private String statusDetail;

}
